docs/rfcs/0003-etag-changes.md

```diff
@@ -24,54 +24,153 @@ While the current Etag implementation allows for easily adding and removing Etag
+The final option is to not use the request/response to generate an Etag, but instead to use something like the id generator to generate a random identifier for an etag.  This could avoid issues with differences in request/response but create potential problems around collisions.  However, this might not be an issue as it's unlikely to occur and even if it did, there would be a very low chance of using these commmon identifiers to update different resources.
+
+Geenrating the Etag could also be combined with data to indicate versioning (i.e. create has v1 encoded, first update v2 etc.) but care must be taken to make sure that these Etags look truly random as it's possible users begin ascribing meaning to the Etag values if they are too similar or seemingly follow a convention.
+
+## Recomendations
 
-### Other considerations
+Based on the information discussed above, these are what the new Etags process should do:
+
+- The Etag should be stored in the database under a separate `etag` table
+- While the distributed cache would be useful, as an initial implementation it would be ok to run using the direct database calls.  However, this should be monitored and if it seems that the database begins to become resource starved, then the distributed cache should be imlemented.  However, in the intervening time between deciding the cache is needed and deploying the implementation, additional resources can be dedicated to the [RDS instance](https://aws.amazon.com/rds/instance-types/) to mitigate issues with resource starvation.
+- The hash of the response body should be used to generate the Etag as the S3 object has too many instances where it's not updated
+
+## Other considerations
 
 based on the [API scratchpad tests](https://github.com/dlcs/iiif-presentation-tests/blob/04213f185bf4fb370855e7e37be27ee4587234bf/tests/apiscratch/t0071-update-managed-asset-manifest.spec.ts#L60-L62) shows that editable resources (i.e.: PUT/POST) should not respond with Etags and instead would require a GET to retrieve the Etag.  This would have implications on the automated tests.
 
-The presentation API [sends Cache-Control: no-cache headers](https://github.com/dlcs/iiif-presentation/issues/140) to stop clients from caching values and then serving old Etags.  As it stands, this shouldn't have an impact on how the Etags work
\ No newline at end of file
+The presentation API [sends Cache-Control: no-cache headers](https://github.com/dlcs/iiif-presentation/issues/140) to stop clients from caching values and then serving old Etags.  As it stands, this shouldn't have an impact on how the Etags work
```

Comment:
Does `no-cache` have any affect on how CDNs or reverse proxies (ie CloudFront) work? Or does including etag mean it'll make requests to see if content expired?